app.js

```diff
@@ -43,6 +43,7 @@ router.get('/people', function(req, res) {
         {name: 'swparkz', url: 'assets/swparkz.png'},
         {name: 'gsongsong', url: 'assets/sjeon.jpg'},
         {name: 'tak0ya', url: 'assets/tak0ya.jpg'}
```

Comment:
Can you add a comma at the end of Line 45?

Reply:
Dear Professor Chae,

I checked my code is working from localhost. 
And I add and commit as explained on the course website. 

It goes well except at push level with this comment.

"Permission to sein92/projectx.git denied to starbuckscoder"

I ignored it and I moved forward by requesting merge. Now I manually "added a comma, added my photo to the file"
Could you cast a light on my procedure?

Reply:
One difference I noticed was we were supposed to clone from "wochae/projectx.git" instead of "sein92/projectx.git". Other than this, I could not see what went wrong. One thing I can recommend was to use "gitk" or similar tools to review the changes before push :)